Dockerfile.prometheus

```diff
@@ -1,31 +1,24 @@
 FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_golang_1.22 as builder
-
 RUN dnf module install -y nodejs:16/development
 ENV NPM_CONFIG_NODEDIR=/usr
+WORKDIR /opt/app-root/src
 
-RUN yum install -y --setopt=tsflags=nodocs bzip2 jq golang-github-prometheus-promu
+RUN yum install -y --setopt=tsflags=nodocs bzip2 jq
 
 COPY prometheus/ .
 
-WORKDIR /workspace
-
 ENV GOFLAGS='-mod=mod'
-ARG GO_BIN=ARG GO_BIN=/workspace/prometheus/deps/gomod/bin
-
-# Move the prometheus-promu we install to go's bin/ to avoid re-installing it during the build
-# Makefile.common expects the promu binary in a certain location
-RUN mkdir $GO_BIN \
-    && ln -s $(which promu) $GO_BIN/promu \
+RUN go install -mod=mod github.com/prometheus/promu@v0.15.0 \
```

Comment:
Let's use `v0.17.0`.
We can also just grab the pre-build binary from https://github.com/prometheus/promu/releases/tag/v0.17.0. maybe that also avoids the `WORKDIR /opt/app-root/src` change?

Reply:
Hmmm note that we would need a way to tell konflux about the architecture of the change. Let's move this to 0.17 and put a different PR later on so we can unblock the CI

Reply:
Not just this?
```
ARG TARGETARCH
ARG TARGETOS
RUN wget https://github.com/prometheus/promu/releases/download/v0.17.0/promu-0.17.0.${TARGETOS}-${TARGETARCH}.tar.gz
```